scripts/asm-installer/install_asm

```diff
@@ -566,6 +569,17 @@ can_register_cluster() {
   fi
 }
 
+can_create_namespace() {
+  if ! can_modify_at_all; then false; return; fi
+
+  if [[ "${ENABLE_ALL}" -eq 1 || "${ENABLE_NAMESPACE_CREATION}" -eq 1 ]] \
+    && ! is_managed; then
```

Comment:
I think we still need the namespace created even with MCP, we can drop the `&& ! is_managed`.

Reply:
Updated. I included this because of [this line](https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/install_asm#L240).